utils/game-utils.js

```diff
@@ -1,11 +1,26 @@
+/**
+ * @returns int <= 0 if card1 is lower rank or same rank & lower suit
+ * @param {rank: number, suit: number} card1
+ * @param {rank: number, suit: number} card2
+ */
+function compareByRankThenSuit(card1, card2) {
+  // Compare by suit (clubs->diamonds->hearts->spades)
+  // Then compare by rank (1,2...12,13)
```

Comment:
Comment is backwards--rank is considered first

Reply:
Wait actually I'm confused because the return statement does appear to consider suit first but the function name implies rank is first. I'm not sure the full context yet but for scuttling purposes we sort by rank first and use suit to break a tie. Something's inconsistent here but I'm not sure which way it should go

Reply:
Yeah okay I think the logic and the tests should switch to match the function name (rank then suit) because this function controls the order of cards in your hand and it makes sense to group them by rank more than by suit